package.json

```diff
@@ -38,6 +38,7 @@
     "parse-server-push-adapter": "1.0.4",
     "parse-server-s3-adapter": "1.0.4",
     "parse-server-simple-mailgun-adapter": "1.0.0",
+    "parse-aws-firehose-logger-adapter": "acinader/parse-aws-firehose-logger-adapter",
```

Comment:
👍 

The adapterLoader support many many ways to pass the loaded adapter:

- 'materialized adapter' 
    `let adapter = new MyAdapter();`
- option hash
    `let options  = {key: 'value'} => new DefaultAdapter(options)`
- options with module:
   `let options = { module: 'path-to-module', options: { option1: true} }`
   => `let Adapter = require(options.module); let adapter = new Adapter(options.options)`
- path to module
   `let module = './my_logger.js'; => let adapter = require(module);`